po/fr.po

```diff
@@ -35,7 +35,7 @@ msgstr "N°"
 #: ../addons/src/ao_bookmarklist.c:350 ../devhelp/src/dhp-object.c:494
 #: ../devhelp/devhelp/dh-window.c:1142
 msgid "Contents"
-msgstr "Contenu"
+msgstr "Contenus"
```

Comment:
Doubt it's really the plural form in English, but rather the whatever-it-is-named fact this word is always plural when referring to actual stuff.
Would have to triple check in the context of the plugins, but I think singular form in French is the right choice.